baybe/utils/chemistry.py

```diff
@@ -88,13 +88,10 @@ def _smiles_to_mordred_features(smiles: str) -> np.ndarray:
     """
     try:
         return np.asarray(
-            _mordred_calculator(Chem.MolFromSmiles(smiles)).fill_missing(),
-            dtype=DTypeFloatNumpy,
+            _mordred_calculator(Chem.MolFromSmiles(smiles)).fill_missing()
```

Comment:
Just for me to understand: your rational is to say "conversion only happens in the baybe objects, not utilities"?

Reply:
no in this file I shifted the DTYPE conversion to the next function below which sues this one here, strangely when the conversion was done here it did not always have correct type
this delete is just there because the other function without the underscore does the conversion now

Reply:
But that's what I meant: if someone were to use the utility directly, we say we don't bother about dtypes, only when used in "actual" baybe context. Have no strong opinion on what is better though...

Reply:
hmm didnt think about that, the utility that is now potentially not in the right dtype is the private one with `_` anyway, the none private one has the right dtype so imo this is reasonable

Reply:
OK, I guess there is no obvious answer anyway, so let's just keep until we run into problems 👍🏼